.yarnrc

```diff
@@ -0,0 +1,5 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+yarn-path ".yarn/releases/yarn-1.22.10.cjs"
```

Comment:
pins yarn version to prevent lockfile corruption (happens from time to time when mixing yarn 1.x versions)